system_files/deck/shared/usr/share/ublue-os/just/60-custom.just

```diff
@@ -8,6 +8,12 @@ toggle-autologin:
       sudo touch $DESKTOP_AUTOLOGIN
     fi
 
+# Install Bazzite's Steam Game Mode Startup Video
+install-gamemode-video:
+    mkdir -p $HOME/.steam/root/config/uioverrides/movies/deck_startup.webm
```

Comment:
Shouldn't this just be 

`mkdir -p $HOME/.steam/root/config/uioverrides/movies/` ?